lib/portage/util/__init__.py

```diff
@@ -126,8 +133,8 @@ def writemsg_stdout(mystr: str, noiselevel: int = 0) -> None:
 def writemsg_level(msg: str, level: int = 0, noiselevel: int = 0):
```

Comment:
Shouldn't we annotate this function while we're here? It's very similar to the other.

Reply:
Did you mean to add return type to `writemsg_level`? My bad, just did. Pushing it.